scene/resources/curve.cpp

```diff
@@ -445,10 +445,10 @@ void Curve::set_bake_resolution(int p_resolution) {
 	_baked_cache_dirty = true;
 }
 
-real_t Curve::interpolate_baked(real_t offset) {
+real_t Curve::interpolate_baked(real_t offset) const {
 	if (_baked_cache_dirty) {
 		// Last-second bake if not done already
-		bake();
+		const_cast<Curve *>(this)->bake();
 	}
```

Comment:
Is this safe behavior? Seeing bake changes the data within the object it's not constant. Is there a reason for making this function constant?

Reply:
Its an internal cache, so its either this or making it mutable and I did not want to change a lot of code.